### ISSUE TITLE
tastykitchen data quality

### ISSUE THREAD

**Issue by famagusta** [fictive-kin/openrecipes#214]
I have noticed that the data for tasty kitchen has incorrectly crawled the ingredients attribute for most of the data. 

e.g. (the first entry)
{"description": "Empanadas made with homemade vegan dough, filled with melty mozzarella, fresh tomatoes and basil.", **# "ingredients": "1-\u00be ounce, weight 1-\u00be ounce, weight\n2 cups 2 cups\n\u00bc teaspoons \u00bc teaspoons\n\u00bd cups \u00bd cups\n10 whole 10 whole\n5 ounces, weight 5 ounces, weight\n\u00bd cups \u00bd cups\n1 whole 1 whole"**, "url": "http://tastykitchen.com/recipes/appetizers-and-snacks/caprese-empanadas/", "image": "http://static.tastykitchen.com/recipes/files/2013/03/Caprese-Empanadas-by-clem-on-March-15-2013-410x273.jpg", "ts": {"$date": 1365299590596}, "datePublished": "2013-03-15", "source": "tastykitchen", "recipeYield": "10", "_id": {"$oid": "5160d18696cc620d2615320c"}, "cookTime": "PT20M", "prepTime": "PT15M", "name": "Caprese Empanadas"}

**Comment by funkatron** [comment]
Yes, that seems to be the case. Our time has been extremely limited to work
on this, but if you have a chance to look into the crawler and identify the
issue, that would help a lot.

**Comment by famagusta** [comment]
Sure - will take a peek under the hood. Overall the project is a great resource.

**Comment by famagusta** [comment]
there is already a pull request with the solution
